Button.css

```diff
@@ -0,0 +1,129 @@
+@import "/ComponentLibrary/Css/main.css";
+:root{
+    /* Colors  */
+     --primary-color:#420039;
+     --secondary-color:#E1DEE9 ;
+     --plain-primary:white;
+     --danger:red;
+     --success:green;
+     --primary-back:skyblue;
+     --primary-text:blue;
+     --secondary-back:rgb(185, 180, 180);
+     --secondary-text: black;
+     --warning-back:rgb(248, 221, 197);
+     --warning-text:rgb(63, 47, 5);
+     --danger-back: pink;
+     --danger-text:rgb(134, 6, 6);
+     --success-back:rgb(181, 224, 241);
+     --success-text:green;
+ 
+    /* Typography */
+    --designed-font: 'Kaushan Script', cursive;
+ 
+ }
```

Comment:
same!

Reply:
Thanks nikhil I'll take care of those above mentioned points